concepts/observability-strategy/assets/strategy-current.drawio.svg

```diff
@@ -0,0 +1,523 @@
+<svg host="65bd71144e" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="821px" height="742px" viewBox="-0.5 -0.5 821 742" content="&lt;mxfile&gt;&lt;diagram id=&quot;Foci5szlibi6Nbq_EEBP&quot; name=&quot;Page-1&quot;&gt;7Vtbc6M2FP41fowHIQT40bGddNvdzk7TnXYfZSPbdDFyhbyx++srQGBAIvYGgWeStWdiOLqAvnPRdyRlBGe74yPD++0nGpBoZFvBcQTnI9u2LQTFTyo5FRIL5JINC4NcVhE8hf8RKbSk9BAGJKlV5JRGPNzXhSsax2TFazLMGH2uV1vTqP7UPd4QRfC0wpEq/SsM+DaX+rZ3lv9Cws22eDJwJ3nJDheV5UiSLQ7oc0UEFyM4Y5Ty/Gp3nJEoRa/AJW/30FJavhgjMb+mgZM3+I6jgxybfC9+KgbL6CEOSFrfGsH7523IydMer9LSZ6FfIdvyXSTugLhch1E0oxFlWVv4kH2EPOGMfiOVEjf7pC1ozCvy/CPk6kjk4L4TxsmxIpIjeyR0Rzg7iSqy9M6XKEs7uwMTKXiuak3KthWNuUVFLC1lU3Z+BlNcSDz12LqGsQ1wss3qaoCeo4U/d3RAW9nHEKCoCaijAup5GkAdZABQNCCgCzf9Dg0otK/EE7gG8PSvALTFt2MakxICZbwaVNohcJo2BVQMyuFWMUCeAQyA6qXT/T4KV5iHNB4oGNrQcVCgBkOjpmY3TQ2ppqZD2QTIEwXk3047LCRPp4ST3UAwl3D2CTMAY9+bVD812JGKOrDg2EOaScgA8rbfhvwsOgjomQK9GCev41vHUXq+JhjgKNzE4nYl0BMdw/sUNeFI0VQW7MIgSB+jVWhd5X1qyG0GHE8NOI7GE2wT+tB7wl3hB9bveEeSDJl3pxhkocvsArgaTzGhGWgpmvmSpP7xrlWC6ipxXI1GQE++goACOAlEHiZvKeNbuqExjhZnaQOac52PlO6ljv4hnJ9kUokPnNY1SI4h/zttPkby7mulZH6UPWc3p+ImFkOrNEpvv1bLzs2yu6JdY+Za4pUfwBcJQqtOE3pgK4lSkV9zzDakMZmkCL6oeUYiQX2+1zNdnRpl0880FK9SWgxoOLFjNSwhfynZqmEM5WtcZx9Q8dgPiWBtI9uNUrdcsprxuP8e0qw6c5e7JFP/VFTw98cM16JYXG3S38+MHk8diYl4krQz/1UpW6/B9iIbhD2xQaTOge+Dcpe+MADldtXp7KdzvNI5dHrryzlcDWE/LAmLCSfJ20yYrLFrtydMdzoSaIGx3VPG5KrRSfLATzjGG4Gxbf1B5HRrVA0o/b5u9u+wJmpr8dUt4RkA11OjkkxHewY3QMQPtPHEt5dQtxBt0sbhTTG/DY0uKXGNEJ/58XWUmAChOe/F2JSXFPswToXCF6S98uRWCt+q5QZhrrNqX+74VNl3vqralVhPGcOnSoV9SpiTSs9N3t1YRgVyRenhdfXFRf4GbSy/uYaC6uPJAVFYvtJRuahfvAboLV0AQM0XPtJNUmVEObv5k4moopF/Eg8LV4nWnT7iJYnqLnB9js+IoFx4mfWXGqNUtugc3Y/Q/KWFdrnFKRuPysS7asnehSnBGgPPdxqzbn772myw6AZO6k3oep2Qrpr01TX7D7EIA4edGLI+iXjFSo2IMffZ1/ByTasqf2DPxB03/OZOs7M5sccajgQNTClF2LvpxhFQdncH3TjyVaIuxhgJX3z79qcQGo316YA3YnsqPx/e9hxl/tPYHtQxOiO2N1FpdDFbJXscF/PVE6csPcmSzgku3qWmkP8Vkqkgc6ckLGc58chq0zduwI532YCBLsU3YcEaHvKZpd1syUElF12ynvV6ba9WOvYcuEsX9Zz1lMe1Tg1AL20eGNn4VDnCg7iK+X3IjWIcYOKvtRi7K58s18NirNte7gviyc/EcnRtYqlkkRe2cAqGVUs2J3pT6DexhO6PJZYX6l9KLBvMAjQP4l2bWJYN2zoymViq56jeR2I5uZhY2p7lNpjSqFNiKbv26p0W0d5omjlRGb6Yp4UCE50OC90anFoWYI4W2hh15aJlt/yqeTBv0FXLiUrzXwA/dzeD0M+nC/9hdivomxtQw0Jfdnwd9m3Bq+OWlD27GfzQhWN0UwUAnQLKDKGpgWkkRrvLNlCYUTV0zh86qQE18weNDvyeuC2wbM2c/i00Cm/n1KETvK51S3jVHPhXTEzbbzvJn7gexD3br2PfEmDdEm11iejy4Q8AW05/PDK8FtHm2qWjG51NqBwwAbAfl0FDKlRzMCQUmUA3uHsI00OCAtSJsggj1pcPbziWDAmy/Q5idXMdzdbtJRlaSBO353/wzPPS8//JwsX/&lt;/diagram&gt;&lt;/mxfile&gt;">
```

Comment:
Pls fix the spelling:

- `User Managed` => `User-Managed`
- `Kyma Managed`=> `Kyma-Managed`
- `FluentBit` => `Fluent Bit`
- `Storage & Analysis` => `Storage and Analysis`
- `Prometheus Alertmanager` => I believe this should be `Prometheus, Alertmanager` ?